packages/label-sync/src/label-sync.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import { request } from 'gaxios';
+import { GCFBootstrapper } from 'gcf-utils';
+import { GitHubAPI } from 'probot/lib/github';
+import {labels} from './labels.json';
+
+const reposUrl = 'https://github.com/googleapis/sloth/blob/master/repos.json';
+
+interface RepoResponse {
+  repos: {
+    language: string;
+    repo: string;
+  }[]
+}
+
+export = (app: Application) => {
+  // Sync labels when a new repo is created
+  app.on('schedule.repository', async context => {
+    // this needs logic to list all the repos we
+    // have label sync installed on:
```

Comment:
`schedule.repository` will be called on whatever schedule we wish, with an authenticated `context.github` instance. We should update the bot logic to pull a list of repos that need to have their labels fixed (we probably want to do this on a fairly infrequent schedule, e.g., daily).